dll/win32/shell32/CQueryAssociations.cpp

```diff
@@ -98,9 +98,12 @@ HRESULT STDMETHODCALLTYPE CQueryAssociations::Init(
         HRESULT hr;
         LPCWSTR pchDotExt;
 
-        pchDotExt = PathFindExtensionW(pszAssoc);
-        if (pchDotExt && *pchDotExt)
-            pszAssoc = pchDotExt;
+        if (StrChrW(pszAssoc, L'\\'))
```

Comment:
Is `/` not recognized as path separator here?

Reply:
It's backslash only detected.